grammar.js

```diff
@@ -1309,8 +1310,8 @@ module.exports = grammar({
       seq(
         // opchar
         /[\-!#%&*+\/\\:<=>?@\u005e\u007c~\p{Sm}\p{So}]/,
-        // opchar minus slash
-        /[\-!#%&*+\\:<=>?@\u005e\u007c~\p{Sm}\p{So}]/,
+        // opchar minus slash and asterisk
+        /[\-!#%&+\\:<=>?@\u005e\u007c~\p{Sm}\p{So}]/,
```

Comment:
Would this cause issue matching on `**`?

Reply:
Somehow it doesn't: https://github.com/tree-sitter/tree-sitter-scala/pull/251/commits/8f1f347918b6acce1b0718d79c26f2ae9dfb92b9

Reply:
But this change is essential for this PR
Without it the comment start in the following code is parsed as operator_identifier
```scala
object X:
  /** foo */
```

Reply:
I guess `**` would match `repeat(/[\-!#%&*+\/\\:<=>?@\u005e\u007c~\p{Sm}\p{So}]/),` of the `choice` branch.